generators/server/templates/dotnetcore/src/Project/Web/Rest/Problems/ProblemDetailsConfiguration.cs.ejs

```diff
@@ -33,18 +33,18 @@ namespace <%= namespace %>.Web.Rest.Problems {
 
         public void Configure(ProblemDetailsOptions options)
         {
-            options.IncludeExceptionDetails = ctx => _environment.IsDevelopment();
+            options.IncludeExceptionDetails = (ctx , details) => _environment.IsDevelopment();
 
-            options.OnBeforeWriteDetails = details => {
+            options.OnBeforeWriteDetails = (ctx, details) => {
                 // keep consistent with asp.net core 2.2 conventions that adds a tracing value
                 var traceId = Activity.Current?.Id ?? _HttpContextAccessor.HttpContext.TraceIdentifier;
                 details.Extensions["traceId"] = traceId;
             };
 
-            options.Map<AuthenticationException>(exception =>
-                new ExceptionProblemDetails(exception, StatusCodes.Status401Unauthorized));
+            options.Map<AuthenticationException>(exception  =>
+                new StatusCodeProblemDetails(401));
             options.Map<NotImplementedException>(exception =>
-                new ExceptionProblemDetails(exception, StatusCodes.Status501NotImplemented));
+                new StatusCodeProblemDetails(501));
 
```

Comment:
Can you add this ``` 
// Because exceptions are handled polymorphically, this will act as a "catch all" mapping, which is why it's added last.
            // If an exception other than NotImplementedException and HttpRequestException is thrown, this will handle it.
            options.MapToStatusCode<Exception>(StatusCodes.Status500InternalServerError);
```

To catch all exceptions 

You can also use `options.MapToStatusCode` method instead of just map 
(https://github.com/khellang/Middleware/blob/master/samples/ProblemDetails.Sample/Program.cs)

Reply:
This will be done in a minute !